### DISCUSSION TITLE
Kubernetes + Docker + Planet Question

### DISCUSSION THREAD

**Discussion by Strvm** [mediagis/nominatim-docker#495] [Q&A]
Hi,

I am trying to use this project inside a Kubernetes cluster for multiple planet servers. I was wondering if it was possible to dump the postgresql database of an already running server into maybe an S3/Cloud storage and then pull that database everytime the docker is started. 

Would this be a good idea? Has anyone else tried something to speed up the building process when first running the docker? Is it possible to connect to external database?

Thank you

**Comment by leonardehrenfried** [comment]
While it's not the focus of this image, someone made it work with an external postgres instance: https://github.com/mediagis/nominatim-docker/issues/245#issuecomment-1072205751

I have no advice about anything to do with k8s.
